src/Factory.php

```diff
@@ -420,6 +437,26 @@ function ($owningModel, $relationshipName) use ($model) {
         return $this;
     }
 
+    /**
+     * Process any closures added to the afterCreating collection
+     *
+     * The created model will be passed into the closure as the first param
+     *
+     * @param \Illuminate\Support\Collection $result
+     */
+    protected function processAfterCreating(Collection $result)
+    {
+        if ($this->afterCreating->isEmpty()) {
```

Comment:
Is the conditional necessary?

Reply:
Its not needed, was just saving a loop, but I will remove it